src/main/java/AddToDoCommand.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Class to represent add To-Do command.
+ */
+public class AddToDoCommand  extends Command {
+
+    /**
+     * Constructor.
+     * @param inputCommand command inputted by user.
+     * @param isExit whether to exit the chat bot or not.
+     */
+    public AddToDoCommand(String inputCommand, boolean isExit) {
+        super(inputCommand, isExit);
+    }
+
+    /**
+     * Execute the given task.
+     * @param ui the Ui class in charge of aesthetics.
+     * @param list tasklist that stores all the task.
+     * @param storage storage that writes to file.
+     * @throws DukeException exception.
```

Comment:
It's better to specify the condition that will result in the exception, instead of just stating that it is an exception.